kernel/api/src/main/java/org/sakaiproject/event/api/UsageSession.java

```diff
@@ -35,14 +35,14 @@
 	/** String constants for major browser types */
 	public static final String MAC_IE = "Mac-InternetExplorer";
 	
-	/*
+	/**
 	 * @deprecated  discontinued 2008
-	 */
+	 **/
```

Comment:
I don't see any reference online about using two asterisks in the closing tag.  Oracle documents the use of two to open, and one to close.

https://docs.oracle.com/javase/9/docs/specs/doc-comment-spec.html

Reply:
OK